docs/notebooks/atmodel_tutorial.ipynb

```diff
@@ -0,0 +1,1061 @@
+       " (4,\n",
+       "  '0.014*\"action\" + 0.011*\"control\" + 0.010*\"policy\" + 0.009*\"q\" + 0.009*\"reinforcement\" + 0.008*\"optimal\" + 0.006*\"dynamic\" + 0.005*\"robot\" + 0.005*\"environment\" + 0.005*\"reward\"'),\n",
+       " (5,\n",
+       "  '0.015*\"representation\" + 0.012*\"layer\" + 0.011*\"image\" + 0.009*\"object\" + 0.008*\"component\" + 0.006*\"face\" + 0.006*\"map\" + 0.006*\"signal\" + 0.005*\"code\" + 0.005*\"activity\"'),\n",
+       " (6,\n",
+       "  '0.013*\"speech\" + 0.012*\"classifier\" + 0.012*\"class\" + 0.010*\"recognition\" + 0.009*\"mixture\" + 0.009*\"classification\" + 0.007*\"tree\" + 0.007*\"likelihood\" + 0.006*\"node\" + 0.006*\"sample\"'),\n",
+       " (7,\n",
+       "  '0.024*\"neuron\" + 0.021*\"cell\" + 0.010*\"response\" + 0.009*\"spike\" + 0.009*\"stimulus\" + 0.008*\"activity\" + 0.008*\"synaptic\" + 0.006*\"signal\" + 0.006*\"frequency\" + 0.006*\"cortex\"'),\n",
+       " (8,\n",
+       "  '0.019*\"image\" + 0.011*\"chip\" + 0.010*\"motion\" + 0.010*\"circuit\" + 0.010*\"field\" + 0.008*\"analog\" + 0.008*\"direction\" + 0.007*\"visual\" + 0.007*\"map\" + 0.007*\"object\"'),\n",
+       " (9,\n",
+       "  '0.010*\"net\" + 0.009*\"recognition\" + 0.009*\"word\" + 0.009*\"hidden\" + 0.008*\"architecture\" + 0.008*\"character\" + 0.007*\"recurrent\" + 0.007*\"layer\" + 0.007*\"rule\" + 0.007*\"hidden_unit\"')]"
+      ]
+     },
+     "execution_count": 67,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.show_topics(num_topics=10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These topics are by no means perfect. They have problems such as *chained topics*, *intruded words*, *random topics*, and *unbalanced topics* (see [Mimno and co-authors 2011](https://people.cs.umass.edu/~wallach/publications/mimno11optimizing.pdf)). They will do for the purposes of this tutorial, however.\n",
+    "\n",
+    "**TODO:** re-write the interpretation of the topics below, if necessary.\n",
+    "\n",
+    "Below, we use the `model[name]` syntax to retrieve the topic distribution for some authors. Comparing the authors' topics with the topics above, we observe that the model has correctly identified that Yann LeCun and Geoffrey E. Hinton both have something to do with neural networks (topic 5), speech recognition (topic 1 and 5) and statistical machine learning (topic 9). We also observe that Yann LeCun has been particularly occupied with image processing, and perhaps that Geoffrey E. Hinton has worked with visual perception in neuroscience (this is less clear).\n",
+    "\n",
+    "Similarly, Terrence J. Sejnowski and James M. Bower are both neuroscientist, first and foremost, and their topic distributions seem to reflect that."
```

Comment:
Can you print the top 2 words of the topic in this cell? It will save me having to scroll to the cell above?